plugins/transformer/basic/docs/overview.md

```diff
@@ -3,6 +3,7 @@ This CloudQuery transformer plugin provides basic transformation capabilities:
 - Removing columns
 - Adding literal string columns
 - Obfuscating string columns
+- Renaming tables using a name template
```

Comment:
Maybe add the list of supported template variables here too (`{{.OldName}}` is the only one correct?)

Reply:
Changed to:

```
- Renaming tables using a name template (use `{{.OldName}}` to refer to the original name, see example below)
```